Libplanet.Explorer/GraphTypes/BlocksQuery.cs

```diff
@@ -116,6 +153,56 @@ public BlocksQuery(BlockChain<T> chain)
             }
         }
 
+        private IEnumerable<Transaction<T>> ListTransactions(
+            Address? signer, Address? involved, bool desc, long offset, int? limit)
+        {
+            Block<T> tip = _chain.Tip;
+            long tipIndex = tip.Index;
+
+            if (offset < 0)
+            {
+                offset = tipIndex + offset + 1;
+            }
+
+            if (tipIndex < offset || offset < 0)
+            {
+                yield break;
+            }
+
+            Block<T> block = desc ? _chain[tipIndex - offset] : _chain[offset];
+
+            while (limit is null || limit > 0)
+            {
+                int index = desc ? block.Transactions.Count() - 1 : 0;
+                while (index >= 0 && index < block.Transactions.Count())
+                {
+                    Transaction<T> tx = block.Transactions.ElementAt(index);
+                    if (IsValidTransacion(tx, signer, involved))
+                    {
+                        yield return tx;
+                        if (!(limit is null))
```

Comment:
According to [docs][docs], it doesn't look like you need to check null.

[docs]: https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/nullable-types/using-nullable-types#operators

```suggestion
                        limit--;
                        if (limit <= 0)
                        {
                            break;
                        }
```